### DISCUSSION TITLE
How how run  document_exporter in docker container

### DISCUSSION THREAD

**Discussion by stepanov1975** [jonaswinkler/paperless-ng#1264] [Support]
Hi. I am new to docker so sorry if it is a stupid question
I am trying to run document_exporter in a docker container like this:
`sudo docker exec -it paperless-ng document_exporter /data/export`
I am getting error:
`OCI runtime exec failed: exec failed: container_linux.go:367: starting container process caused: exec: "document_exporter": executable file not found in $PATH: unknown`
I am running container like this:

```
docker run -d \
--name=paperless-ng \
-e PUID=1034 \
-e PGID=100 \
-e TZ=Asia/Jerusalem \
-e PAPERLESS_TIME_ZONE=Asia/Jerusalem \
-e PAPERLESS_OCR_LANGUAGES=heb \
-e PAPERLESS_OCR_LANGUAGE=eng+heb \
-e DOCKER_MODS=linuxserver/mods:papermerge-multilangocr \
-e OCRLANG="heb" \
-p 8008:8000 \
-v /volume1/docker/paperless:/config \
-v /volume1/paperless_data:/data \
-v /etc/localtime:/etc/localtime:ro \
-v /etc/timezone:/etc/timezone:ro \
--restart unless-stopped \
linuxserver/paperless-ng:amd64-latest
```

What am I doing wrong?
THanks.

**Comment by Tooa** [comment]
You are not running our official paperless-ng docker image (`linuxserver/paperless-ng:amd64-latest`). You also mixed configuration from paperless and papermerge (`OCRLANG="heb"`, `DOCKER_MODS=linuxserver/mods:papermerge-multilangocr`).

Please try again with our official paperless-ng image. You can either install it as docker with our installation script ([documentation](https://paperless-ng.readthedocs.io/en/latest/setup.html#install-paperless-from-docker-hub-using-the-installation-script)) or you can skip that part and do a [custom docker install](https://paperless-ng.readthedocs.io/en/latest/setup.html#install-paperless-from-docker-hub) yourself.

Once that works for you, head over to our documentation that describes the backup process [here](https://paperless-ng.readthedocs.io/en/latest/administration.html#making-backups) and read it. You are especially interested in the [document exporter section](https://paperless-ng.readthedocs.io/en/latest/administration.html#utilities-exporter).

Regarding your question: 

Looks like the `document_exporter` command is not on the PATH for your `linuxserver/paperless-ng` image. Your docker run command also does not contain a host mount for export. 

```bash
-v /volume1/docker/paperless:/config \
-v /volume1/paperless_data:/data \
```

I can not tell you how that should look like, because it depends on the installation inside the docker container. Since you don't use our container build, the paths may vary. If you still like to use the `linuxserver/paperless-ng` image, you need to head over to the LinuxServer Discord channel and ask for help there.

Hope that helps.

**Reply by stepanov1975** [reply]
@Tooa Thank you for the replay. I am new to docker. I am running the container on Synology NAS. As I understand there is a problem with docker composer on Synology NAS. So is it a way to run the official paperless-ng docker image using the "docker run" command without using composer?
